README.md

```diff
@@ -248,6 +248,15 @@ sudo yum config-manager --set-enabled PowerTools
 sudo yum install pcsc-lite-devel
 ```
 
+On Windows:
+
+No prerequisites are needed. Although it's a good idea to replace the default driver
+with the [YubiKey Smart Card Minidriver](https://www.yubico.com/products/services-software/download/smart-card-drivers-tools/).
```

Comment:
Could you elaborate on why is it a good idea? Will users experience errors otherwise?

Reply:
I'm was not sure as Yubico states on their website their driver adds additional features like pin change and so on. I've uninstalled the driver and ran the tests with the default driver by Microsoft and everything seems to work. I rephrased the paragraph to state that the default driver looks fine for the unittests but maybe the additional driver is needed for advanced usage.